src/GapicClientTrait.php

```diff
@@ -779,6 +810,10 @@ private function createCallStack(array $callConstructionOptions)
         }
         $callStack = function (Call $call, array $options) {
             $startCallMethod = $this->transportCallMethods[$call->getCallType()];
+            if (!$this->transport) {
+                // Create transport at call-time to allow for lazy initialization
+                $this->transport = $this->buildTransport();
+            }
             return $this->transport->$startCallMethod($call, $options);
```

Comment:
```suggestion
            return $this->getTransport()->$startCallMethod($call, $options);
```

Reply:
I forgot that this was done intentionally to prevent breaking changes in the event that someone overrode the getTransport method in a client subclass.